go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/sigstore/sigstore v1.8.3
 	github.com/spf13/cobra v1.8.0
 	github.com/stretchr/testify v1.9.0
+	github.com/tzvetkoff-go/fnmatch v0.0.0-20220210160758-879480b5e662
```

Comment:
why this dependency instead of similar ones that appear to be more widely used?

- https://github.com/bmatcuk/doublestar
- https://github.com/danwakefield/fnmatch

Reply:
doublestar is _way_ nicer, switched!

Reply:
also evaluating danwakefield/fnmatch.